Assets/Scripts/Network/NetworkController.cs

```diff
@@ -88,21 +88,26 @@ public void SetNetworkAddress(string destination)
     private void Awake()
     {
         DontDestroyOnLoad(this.gameObject);
-        InsureUsername();
+        EnsureUsername();
     }
 
-    void InsureUsername()
+    void EnsureUsername()
```

Comment:
thank you for fixing this! I don't even remember seeing this line of code but now I see it, it pains me! You have ensured that I remain sane.